eslint.config.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  plugins: [
+    "@typescript-eslint",
+    "header"
+  ],
+  extends: [
+      "./node_modules/gts",
```

Comment:
I wonder if we could extend the eslint config from the main repo so that changes are automatically picked up after a main repo release? wdyt @dyladan?

Reply:
I wonder if there is a way to depend on the file directly in github instead of a release. Very interesting idea though. A package like `@opentelemetry/eslint-config` would be a welcome addition I think.

Reply:
Yes we could do something like `npm install --save-dev https://github.com/opentelemetry-js/opentelemetry-js/tarball/master`

Reply:
@dyladan maybe we can create some general package that can be easily shared between contrib repo and here andd maybe even a 3rd party contrib. We could put there a bit more stuff, eslint, karma config, etc. basically all things that could be shared. WDYT ?